docker/src/main/java/clocker/docker/location/DockerContainerLocation.java

```diff
@@ -188,12 +190,13 @@ public int execScript(Map<String,?> props, String summaryForLogging, List<String
         for (String commandString : filtered) {
             parseDockerCallback(commandString);
         }
-        if (getOwner().config().get(DockerContainer.DOCKER_USE_SSH)) {
+        boolean entitySsh = Boolean.TRUE.equals(entity.config().get(DockerContainer.DOCKER_USE_SSH));
+        boolean dockerSsh = Boolean.TRUE.equals(getOwner().config().get(DockerContainer.DOCKER_USE_SSH));
+        if (entitySsh && dockerSsh) {
```

Comment:
You say it might be set on the clocker blueprint itself: but it seems that you *require* it to be set on the clocker blueprint itself, otherwise `entitySsh && dockerSsh` will not both be true. Is that right?